README.md

```diff
@@ -104,3 +104,103 @@ to a Kubernetes Cluster.
 | <a name="output_iam_role_name"></a> [iam\_role\_name](#output\_iam\_role\_name) | Name of IAM role |
 | <a name="output_iam_role_path"></a> [iam\_role\_path](#output\_iam\_role\_path) | Path of IAM role |
 | <a name="output_iam_role_unique_id"></a> [iam\_role\_unique\_id](#output\_iam\_role\_unique\_id) | Unique ID of IAM role |
+
```

Comment:
Can you delete the ones above? They are repeated.

Reply:
The README still has duplicate tf-docs generated stuff @franklinpashok

Reply:
remove the duplicates in readme. yes the tf-docs if run after deleting it updated again. Now fixed